examples/run-blockchain/index.js

```diff
@@ -17,14 +17,24 @@ var blockchainDB = levelMem()
 
 var state = new Trie()
 
-var blockchain = new Blockchain(blockchainDB)
-blockchain.ethash.cacheDB = level('./.cachedb')
+var validate = false
+
+var hardfork = 'byzantium'
+
+var blockchain = new Blockchain({ db: blockchainDB, hardfork, validate })
+
+if (validate) {
+  blockchain.ethash.cacheDB = level('./.cachedb')
+}
 
 var vm = new VM({
   state: state,
-  blockchain: blockchain
+  blockchain: blockchain,
+  hardfork
 })
-var genesisBlock = new Block()
+var genesisBlock = new Block({ hardfork })
+
+testData.homestead = true
```

Comment:
I may be missing something, but I think this has no effect. If that's the case, can you remove it?

Reply:
yes, it has not effect. I'll remove it.